service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationProcessors.java

```diff
@@ -150,4 +151,23 @@ public static Processor slackEventProcessor(String routeId, String message) {
       exchange.getIn().setBody(AuditEvent.fromAuditable(auditable, routeId, msg));
     };
   }
+
+  /**
+   * This sets up a skeleton lighthouse HealthDataAsessment object in the event of a timeout from
+   * lighthouse. We know the fields we MUST have for evidence merge from the properties we saved on
+   * the exchange.
+   *
+   * @return
+   */
+  public static Processor lighthouseContinueProcessor() {
```

Comment:
It seems like the new body should capture what when wrong or that something went wrong. Otherwise, how will downstream code distinguish a valid response from LH vs. this skeleton response used for timeouts.

Reply:
In this case, the ask that was the downstream code *doesnt* distinguish. We have to continue processing. the way that anyone else will know this happens is with a slack notification.